ethereum_history_api/oracles/src/noir/oracles/receiptOracle/encode.ts

```diff
@@ -2,16 +2,16 @@ import { type ForeignCallOutput } from '@noir-lang/noir_js';
 import { statusToHex } from '../../../ethereum/receipt.js';
 import { padArray } from '../../../util/array.js';
 import { encodeField, encodeHex, encodeProof, encodeBytes } from '../common/encode.js';
-import { ZERO_PAD_VALUE } from '../common/const.js';
+import { MAX_TRIE_NODE_LEN, ZERO_PAD_VALUE } from '../common/const.js';
 import { Proof } from '../../../ethereum/proof.js';
 import { TransactionReceipt } from '../../../types.js';
 import { BYTES_32_ZERO, U1_ZERO } from '../../../util/const.js';
 
-const MAX_RECEIPT_PROOF_LEVELS = 7;
-const MAX_RECEIPT_LENGTH = 532;
-export const MAX_RECEIPT_RLP_LENGTH = 532;
-const MAX_RECEIPT_PROOF_LENGTH = MAX_RECEIPT_LENGTH * MAX_RECEIPT_PROOF_LEVELS;
-export const KEY_LENGTH = 4;
+export const MAX_RECEIPT_KEY_LEN = 3;
+export const MAX_RECEIPT_KEY_NIBBLE_LEN = 6;
```

Comment:
Maybe directly calculate this value
```suggestion
export const MAX_RECEIPT_KEY_NIBBLE_LEN = 2 * MAX_RECEIPT_KEY_LEN;
```

Reply:
Hmm that would be good but linter has a problem with 2 being a magic number. Not sure if I export 2 to some const like NIBBLES_IN_BYTE it will improve readability